installer/x86_64/install.sh

```diff
@@ -20,6 +20,21 @@ _trap_push() {
 }
 _trap_push true
 
+read_conf_file() {
+    local conf_file=$1
+    while IFS='=' read -r var value
+    do
+        f_char=$(echo $var | cut -c1)
+        [ "$f_char" = "#" ] && continue
+        [ -z "$var" ] && continue
+        # remove double quote in the beginning
+        tmp_val=${value#\"}
+        # remove double quote in the end
+        value=${tmp_val%\"}
+        eval "$var=\"$value\""
```

Comment:
Is it possible to use `declare` instead of `eval` ?

Reply:
I am not familiar with `declare`. What is the advantage to use `declare` instead of `eval`?

Reply:
`declare` is safer than `eval`. I know it in bash, not sure about this environment (sh).

---
In reply to: [568218112](https://github.com/Azure/sonic-buildimage/pull/6600#discussion_r568218112) [](ancestors = 568218112)

Reply:
I tried it in sh environment. There is no `declare` command.